src/accessom_coupler/mom_oasis3_interface.F90

```diff
@@ -71,6 +71,18 @@ module mom_oasis3_interface_mod
 !   <DATA NAME="send_after_ocean_update" TYPE="logical" DEFAULT=".FALSE.">
 !    TRUE if coupling strategy requires we send data to coupler AFTER updating the ocean
 !   </DATA>
+!   <DATA NAME="frac_vis_dir TYPE="real" DEFAULT="0.215">
```

Comment:
you're still missing the closing " for these 4 NAMES

Reply:
Aha, it was a misplaced quote, not a doubling up.